supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -601,6 +601,18 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*: xref:content-view[Content View]
 
+discrete]
+[[compute-node]]
+==== image:images/yes.png[yes] compute node (noun)
+*Description*: In Red Hat OpenShift, a "compute node" is a node that is responsible for executing workloads for lucter users. Also known as worker nodes. 
```

Comment:
What is "lucter" supposed to be here? "cluster"? Also, should be italics when introducing the term. Maybe also consider if it's appropriate to italicize worker nodes?

```suggestion
*Description*: In Red Hat OpenShift, a _compute node_ is a node that is responsible for executing workloads for cluster users. Also known as _worker nodes_. 
```

Reply:
typo mistake on my end >> cluster :)